frontend/src/components/containers/ContainersDetailContent.tsx

```diff
@@ -102,10 +100,10 @@ const ContainersDetailContent = ({
 
           <TabPane tab={`Experiment (${experimentRunsIDs?.length})`} key="experiment" style={tabStyle}>
 
-            {containerKindsByID[container.kind] && containerKindsByID[container.kind].is_run_container ?
+            {container && containerKindsByID[container.kind] && containerKindsByID[container.kind].is_run_container ?
```

Comment:
You should not add this test (`container &&`) on the same level as the others since the error message shown does not make sense if the container is just not loaded ...

Reply:
what about this?
```tsx
{container &&
              (containerKindsByID[container.kind] && containerKindsByID[container.kind].is_run_container ?
                <ExperimentRunsListSection experimentRunsIDs={experimentRunsIDs} />
                :
                <div> Experiments are not run directly on containers of kind {container?.kind} </div>
              )
            }
```

Reply:
yep